bucket/openjdk-ea.json

```diff
@@ -1,21 +1,21 @@
 {
-    "description": "Official Early-Access Builds of OpenJDK",
+    "description": "Official production-ready open-source builds of OpenJDK 21",
```

Comment:
Please revert back to early access.

Reply:
Thanks for pointing out, I just forgot about that and copy-pasted from `openjdk21.json`